proposals/simd/W3CTAG-SIMDExplainer.md

```diff
@@ -0,0 +1,70 @@
+# WASM SIMD Web Platform explainer
+
+## User benefits
+Modern processors support running multiple common computations in a single operation, so that instead of multiplying a number with a number, you can add (or multiple, subtract etc) the individual components of a fixed sized vector instead.
+
+##### Scalar operation #####
+
+```
+Ax + Bx = Cx
+Ay + By = Cy
+Az + Bz = Cz
+Aw + Bw = Cw
+```
+
+##### SIMD Operation of Vector Length 4 #####
+
+`Ax`|`Ay`|`Az`|`Aw`
+--|--|--|--
+
+<pre>
+            +
+</pre>
+
+`Bx`|`By`|`Bz`|`Bw`
+--|--|--|--
+
+<pre>
+            =
+</pre>
+
+`Cx`|`Cy`|`Cz`|`Cw`
+--|--|--|--
+
+
+
+
+
+
+This feature is called [Single Instruction Multiple Data (SIMD)](https://en.wikipedia.org/wiki/SIMD), and though there exist hardware that supports vector lengths up to 512 bits, 128-bit SIMD is the most common and is supported across common hardware architectures.
+
+SIMD has driven large speed ups in certain cases such as image manipulation, video encoding/decoding, machine learning, game engines and physics engines etc[3] - with some of these use-cases not being usable without SIMD support, making SIMD support for the web platform essential for achieving near-native speed with certain native applications.
```

Comment:
Perhaps remove the [3] here as it's not clear exactly which part references the blog post, or link it in as with other content to be consistent with the rest of this doc?

Reply:
Added.